test/integration/targets/dms_endpoint/tasks/main.yml

```
@@ -0,0 +1,135 @@
+---
+
+- name: set connection information for all tasks
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      #security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
+  no_log: yes
+
+- block:
+  - name: create endpoints
+    dms_endpoint:
+        state: present
+        endpointidentifier: 'testsource'
```

change reference to fact if decide to use it.
```suggestion
        endpointidentifier: "{{ dms_identifier }}"
```